xcodeproj/internal/output_files.bzl

```diff
@@ -39,8 +39,7 @@ def _get_outputs(*, debug_outputs, id, product, swift_info, output_group_info):
         # TODO: Actually handle more than one module?
         for module in swift_info.direct_modules:
             swift = parse_swift_info_module(module)
-            if swift:
-                break
+            break
```

Comment:
This change is because we want the first direct module always, to potentially get the dependency swiftmodules.